rasa_core/events/__init__.py

```diff
@@ -277,6 +277,7 @@ def _from_parameters(cls, parameters):
             raise ValueError("Failed to parse bot uttered event. {}".format(e))
 
 
+# TODO: DEPRECATED - remove in version 0.10.0
 # noinspection PyProtectedMember
 class TopicSet(Event):
```

Comment:
you also need to fix the `apply_to ` method, it shouldn't do anything

Reply:
done!